service/controller/appcatalog/v1/resource/index/error.go

```diff
@@ -19,3 +19,12 @@ var wrongTypeError = &microerror.Error{
 func IsWrongType(err error) bool {
 	return microerror.Cause(err) == wrongTypeError
 }
+
+var indexNotFound = &microerror.Error{
```

Comment:
```suggestion
var notFound = &microerror.Error{
```

We can use a more generic error here. This helps minimize the number of error types we need to deal with.

Reply:
Note that `Kind` and variable name should be equal. The matcher name should also be aligned. Otherwise it will be a complete mess.